pegasus/sites.v3/code.org/public/about/index.md

```diff
@@ -28,23 +29,26 @@ Code.org&reg; is a non-profit dedicated to expanding access to computer science,
 
 <div style="clear: both;"></div>
 
-## Diversity of students in our courses
-![image](/images/infographics/fit-800/diversity-courses.png)
+## Diversity of students in our courses</h2>
+
+<img src="/images/infographics/fit-800/diversity-courses.png" style="width: 95%;">
```

Comment:
I don't think it will scale for a content editor to have to do this every time they include an image. What is the goal for setting this width? Is there a way to apply this attribute to all `img` types?